.circleci/config.yml

```diff
@@ -143,6 +143,15 @@ jobs:
           command: yarn run test:types
       # - *persist_coverage // we don't run code coverage for integration tests
 
+  bundlesize:
+    <<: *job_defaults
+    steps:
+      - *attach_workspace
+      - run:
+          name: Test types
```

Comment:
`name: Check the size of NGXS bundle`? :D

Reply:
Good catch